src/components/views/Facebook/FacebookLayout.tsx

```diff
@@ -12,7 +14,24 @@ const FacebookLayout = ({ children }: { children: React.ReactElement }) => {
   const currUser = useAppSelector(getCurrUser);
   const navigateTo = useNavigate();
   const [searchParams] = useSearchParams();
+  const [search, updateSearch] = useState("");
   const [query, updateQuery] = useState("");
+  const [filtersClickedOff, setFiltersClickedOff] = useState(false);
+
+  const [advancedFiltersSelected, setAdvancedFiltersSelected] = useState(false);
+  const [name, updateName] = useState("");
+  const [unix, updateUnix] = useState("");
+  const [country, updateCountry] = useState("");
+  const [state, updateState] = useState("");
+  const [city, updateCity] = useState("");
+  const [year27Selected, setYear27Selected] = useState(false);
+  const [year26Selected, setYear26Selected] = useState(false);
+  const [year25Selected, setYear25Selected] = useState(false);
+  const [year24Selected, setYear24Selected] = useState(false);
+  const [professorSelected, setProfessorSelected] = useState(false);
+  const [staffSelected, setStaffSelected] = useState(false);
+  const [studentSelected, setStudentSelected] = useState(false);
+  const [building, updateBuilding] = useState("");
```

Comment:
Having these states stored duplicated at two places doesn't seem ideal.

We can maybe use a global state (like a redux slice that isn't persisted), or manually pass these down as props (would be a lot of painful work IMO)

 @AndrewMuh thoughts?